google_photos_takeout_helper/__main__.py

```diff
@@ -237,7 +241,9 @@ def set_creation_date_from_str(file, str_datetime):
         try:
             # Turns out exif can have different formats - YYYY:MM:DD, YYYY/..., YYYY-... etc
             # God wish that americans won't have something like MM-DD-YYYY
-            str_datetime = str_datetime.replace('-', ':').replace('/', ':').replace('.', ':').replace('\\', ':')[:19]
+            # The replace ': ' to ':0' fixes issues when it reads the string as 2006:11:09 10:54: 1.
+            # It replaces the extra whitespace with a 0 for proper parsing
+            str_datetime = str_datetime.replace('-', ':').replace('/', ':').replace('.', ':').replace('\\', ':').replace(': ', ':0')[:19]
```

Comment:
Generally, this place here needs some extra additional `try-catch`, because all different formats of exif are killing it - but probably somewhere higher up in the script